packages/generator-langium/src/index.ts

```diff
@@ -149,7 +167,17 @@ export class LangiumGenerator extends Generator {
                     'You can add the setup for language tests using Vitest.'
                 ),
                 message: 'Include language tests?',
-                default: true
+                default: true,
+            } as PromptQuestion<Answers>,
+            {
+                type: 'input',
+                name: 'entryName',
+                prefix: description(
+                    'The name of the entry rule in your grammar file.'
+                ),
+                message: 'Your grammar entry rule name:',
+                default: 'Model',
+                when: (answers) => !answers.includeExampleProject
```

Comment:
I think this makes sense with the Hello World example, too. Could you include this variable in the old template?